src/net40/Radical.Windows.Presentation/Messaging/ShowMessageBoxResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Windows;
+
+namespace Topics.Radical.Windows.Presentation.Messaging
+{
+    /// <summary>
+    /// Return value for the Message Box Open Request
+    /// </summary>
+    public class ShowMessageBoxResult
```

Comment:
This entire class can be deleted if the results invocation uses the delegate approach